### ISSUE TITLE
fixed: openjdk support (windows)

### ISSUE THREAD

**Issue by AnzhiZhang** [ggrandes/jentunnel#9]
Hi, could you update this app to support jdk 17? I would really appreciate it.
Related: #2

**Comment by ggrandes** [comment]
The problem seems to be with the system registry issue with openjdk installations (non oracle-jdk), launch4j does not see them. 

This is a small workaround (creating a file.reg and merge on the system with regedit; admin permissions required). Assuming [openjdk-21](https://jdk.java.net/21/) has been installed in the `C:\JAVA\openjdk-8-emulated` emulating 1.8 (very ugly, but works in this case) folder:

```reg
Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SOFTWARE\JavaSoft]

[HKEY_LOCAL_MACHINE\SOFTWARE\JavaSoft\Java Development Kit]
"CurrentVersion"="1.8"

[HKEY_LOCAL_MACHINE\SOFTWARE\JavaSoft\Java Development Kit\1.8]
"JavaHome"="C:\\JAVA\\openjdk-8-emulated"
"MicroVersion"="0"
```

**Comment by ggrandes** [comment]
This issue is solved in [v1.1.2](/ggrandes/jentunnel/releases/)
